### PR TITLE
Allow creation of segment without starTree when HLL columns are providedx

### PR DESCRIPTION
**Pull request by antumbde**

### CODE REVIEW

**Comment by rven1** [comment]
Changes look good. Could you add a test?

**Comment by rven1** [comment]
LGTM

File: pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java [hunk 1]
```diff
@@ -92,7 +92,7 @@ public void testColumnStatsWithoutStarTree() {
     try {
       LOGGER.debug("================ Without StarTree ================");
       helper = new SegmentWithHllIndexCreateHelper(
-          "noStarTree", AVRO_DATA, timeColumnName, timeUnit, "starTreeSegment");
+          "noStarTree", getClass().getClassLoader().getResource(AVRO_DATA), timeColumnName, timeUnit, "starTreeSegment");
```
Comment (rven1) [inline pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java]: Segment name can be noStarTreeSegment

File: pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java [hunk 2]
```diff
@@ -92,7 +92,7 @@ public void testColumnStatsWithoutStarTree() {
     try {
       LOGGER.debug("================ Without StarTree ================");
       helper = new SegmentWithHllIndexCreateHelper(
-          "noStarTree", AVRO_DATA, timeColumnName, timeUnit);
+          "noStarTree", getClass().getClassLoader().getResource(AVRO_DATA), timeColumnName, timeUnit, "starTreeSegment");
```
Comment (rven1) [inline pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java]: noStarTreeSegment ?

File: pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java (hunk 1 above)
Comment (antumbde) [inline pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java]: yes, but does not matter and out of scope of this PR.

File: pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java (hunk 2 above)
Comment (antumbde) [inline pinot-core/src/test/java/com/linkedin/pinot/core/startree/hll/HllIndexCreationTest.java]: out of scope for this PR change.

**Pull request merged**